app/models/manageiq/providers/kubernetes/container_manager/metrics_capture.rb

```diff
@@ -37,8 +37,19 @@ class TargetValidationError < RuntimeError; end
       }
     }
 
+    def fetch_target_authentication
+      ems = target.ext_management_system
+      return unless ems
+
+      auth = ems.authentication_type(:metrics_collection)
+      ems.authentications << Authentication.new(:authtype => :metrics_collection) unless auth
```

Comment:
I'm trying to understand if this (creating an authentication during capture) was what @simon3z and @blomquisg agreed to include and fix later. If so lets add a comment and proceed with it.

Anyway I would think on master we will need to replace it with creating the endpoint on provider addition + migration to handle old providers. Lets discuss this tomorrow.